### DISCUSSION TITLE
Remove repository restriction of the link pull request field

### DISCUSSION THREAD

**Discussion by cilerler** [community/community#11028] [Projects and Issues]
There is a restriction on `Linked pull request`, and it doesn't let me add it from the other repositories.
It becomes an issue, especially when you have 20 repos for a single app,

![image](https://user-images.githubusercontent.com/385958/152589352-ac8e5a61-a9ee-4074-b7fe-5d85b5b8023d.png)

**Comment by Licho1** [comment]
Yes often we need to link issue from another repository to a pull request. Would be nice if it worked at least within single project.
